### DISCUSSION TITLE
Commerce Development - Supported .Net versions

### DISCUSSION THREAD

**Discussion by ahgamal-ms** [microsoft/Dynamics365Commerce.Solutions#319] [Q&A] [answered]
At the current moment the .NET version for the install projects are .NET 4.7.2 and the other CSU (Cloud Scale Unit) projects have targetframework as netstandard2.0.

When or from what version will Microsoft start to support version .NET 8 and/or newer for all the CSU projects, including the install projects?

**Comment by madyke** [comment, marked as answer]
Hi @ahgamal-ms - For .NET Framework, we have discussed moving from 4.7.2 to 4.8.1 as a part of some architecture changes tentatively scheduled for 10.0.42, but that is not committed yet. But it's worth noting that .NET Framework 4.7.2 remains in support with no end of support date even announced yet
[.NET Framework Supported versions](https://dotnet.microsoft.com/en-us/download/dotnet-framework)

For .NET, I'm not aware of any plans to change the target frameworks for CSU projects. But .NET 8 implements .NET Standard 2.1 and earlier, so customers should be free to use newer verions to ensure continued support
[.NET Standard versions](https://dotnet.microsoft.com/en-us/platform/dotnet-standard)
[When to target net8.0 vs. netstandard](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0#when-to-target-net80-vs-netstandard)
